app/scripts/lib/util.js

```diff
@@ -139,6 +139,17 @@ const addHexPrefix = (str) => {
   return `0x${str}`;
 };
 
+const fixHexValue = (str) => {
```

Comment:
Nit: Since we're now throwing an error rather than fixing the value, we should probably rename this. Maybe `validateHexValue` would be better?

Reply:
Also, a JSDoc comment block would be great. They will be required soon, when we update the ESLint config to v9